backend/prod/app.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+const express = require('express');
+const router = require('./router.js');
+const app = express();
+const port = 4200;
+app.use('/', router);
+app.listen(port, () => {
+    console.log(`Example app listening at http://localhost:${port}`);
+});
```

Comment:
the build results should not be under the git. 
under the git, only the files in which we write the code . 
files generated by the machine are not.